### DISCUSSION TITLE
How to clear the recent files?

### DISCUSSION THREAD

**Discussion by ghost** [geany/geany#3662] [Q&A] [answered]
With Geany, there has never been a clear recent files button like in other text editors. On previous versions, everything is put on `geany.conf`, and one has to manually edit this file to delete the recent files list. Now on Geany 2.0, the recent files are no longer stored in `geany.conf`, and I can't find where they are stored either. Like in all previous versions, deleting `recently-used.xbel` has no effect.

**Comment by elextr** [comment, marked as answer]
Settings and session have been separated, session related stuff is now in `session.conf` including recent files.
